flow/record/fieldtypes/__init__.py

```diff
@@ -736,11 +736,12 @@ class command(FieldType):
     executable: path | None = None
```

Comment:
Not specifically touched in the PR, but comment on previous implementation.

Can we not just do `self.executable = path(executable)` and let path field type itself detect if it's windows or posix? would that work?